src/index.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+
+import { PvRecorder } from "@picovoice/pvrecorder-node";
+import { selectPrompt } from "./utils/prompt.js";
+import { playVisualizer } from "./core/visualizer.js";
+import { runCalibration } from "./core/calibration.js";
+
+const DEFAULT_SAMPLE = [200, 3000];
+
+async function main() {
+  const availableMicrophones = PvRecorder.getAvailableDevices();
+  if (availableMicrophones[0] === "NULL Capture Device") {
+    console.log("Sorry, microphone was not detected.");
+
+    return;
+  }
+
+  console.clear();
+
+  const select = await selectPrompt(
+    "Choose 'Sound Calibration' to set up the visualizer for your mic, or 'Skip' to just start!",
+    [
+      { name: "Sound Calibration", value: "calibrate" },
+      { name: "Skip (Use Defaults)", value: "skip" },
+    ],
+  ).run();
+
+  console.clear();
+
+  let volumeSamples = DEFAULT_SAMPLE;
+  if (select === "calibrate") {
+    const calibrationSamples = await runCalibration();
+    const lowest = calibrationSamples[0];
+    const highest = calibrationSamples[1];
+
+    lowest > highest || highest - lowest < 800
+      ? volumeSamples
+      : (volumeSamples = calibrationSamples);
```

Comment:
```suggestion
    volumeSamples = lowest > highest || highest - lowest < 800 ? volumeSamples : calibrationSamples;
```

三項演算子内で変数代入すると読みにくいので

説明用変数を使うとさらに良いかも

```suggestion
    const isFoo = lowest > highest || highest - lowest < 800;
    volumeSamples = isFoo ? volumeSamples : calibrationSamples;
```

Reply:
修正しました！